src/countOccurrences.js

```diff
@@ -18,7 +18,12 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let counter = 0;
+  for (let i = 0; i < phrase.length; i++) {
+    // eslint-disable-next-line eqeqeq
+    if (phrase[i].toLowerCase() === letter) { counter++; }
+  }
+  return counter;
```

Comment:
new line before return
```suggestion

  return counter;
```